includes/admin/class-wc-admin-notices.php

```diff
@@ -64,51 +64,6 @@ public static function init() {
 		}
 	}
 
-	/**
-	 * Parses query to create nonces when available.
-	 *
-	 * @param object $response The WP_REST_Response we're working with.
-	 * @return object $response The prepared WP_REST_Response object.
-	 */
-	public static function prepare_note_with_nonce( $response ) {
```

Comment:
We can't just remove a public static method.

Reply:
Use `_deprecated_function( __CLASS__ . '::' . __FUNCTION__, '5.3.0' );`

Reply:
I thought this was internal enough to sneak away.. but yes you're right, I will deprecate this instead.

Reply:
@claudiosanches - done!